### ISSUE TITLE
Windows build fails - after PR #3966

### ISSUE THREAD

**Issue by planetf1** [odpi/egeria#4116]
The windows builds are currently failing with:
```
FVTUnexpectedCondition{testCaseName='ManageExternalIdsTest', errorDescription='testOneToOne(TestGlossary)', caughtException=InvalidParameterException{parameterName='qualifiedName', reportedHTTPCode=409, reportingClassName='org.odpi.openmetadata.commonservices.ffdc.RESTExceptionHandler', reportingActionDescription='createGlossary', reportedErrorMessage='OMAG-COMMON-409-001 Method createGlossary of service Asset Manager OMAS is not able to create an instance of type null because parameter name qualifiedName is defined as a unique property and value TestGlossary is not available for use', reportedErrorMessageId='OMAG-COMMON-409-001', reportedErrorMessageParameters=[createGlossary, Asset Manager OMAS, null, qualifiedName, TestGlossary], reportedSystemAction='The system is unable to process the request because the unique property for this new entity is not permitted either because it is a reserved value, or it is already in use.', reportedUserAction='Retry the request with a different unique parameter name.', reportedCaughtException=null, reportedCaughtExceptionClassName='null', relatedProperties={parameterName=qualifiedName}}}
FVTUnexpectedCondition{testCaseName='ManageExternalIdsTest', errorDescription='testOneToOne(TestGlossary)', caughtException=InvalidParameterException{parameterName='qualifiedName', reportedHTTPCode=409, reportingClassName='org.odpi.openmetadata.commonservices.ffdc.RESTExceptionHandler', reportingActionDescription='createGlossary', reportedErrorMessage='OMAG-COMMON-409-001 Method createGlossary of service Asset Manager OMAS is not able to create an instance of type null because parameter name qualifiedName is defined as a unique property and value TestGlossary is not available for use', reportedErrorMessageId='OMAG-COMMON-409-001', reportedErrorMessageParameters=[createGlossary, Asset Manager OMAS, null, qualifiedName, TestGlossary], reportedSystemAction='The system is unable to process the request because the unique property for this new entity is not permitted either because it is a reserved value, or it is already in use.', reportedUserAction='Retry the request with a different unique parameter name.', reportedCaughtException=null, reportedCaughtExceptionClassName='null', relatedProperties={parameterName=qualifiedName}}}
```

See 
 * https://dev.azure.com/ODPi/Egeria/_build?definitionId=12 - linux merge builds - all are good
 * https://dev.azure.com/ODPi/Egeria/_build?definitionId=30 - windows merge builds
 * https://dev.azure.com/ODPi/Egeria/_build/results?buildId=14182&view=results - First failed windows merge build

Reported by @yevgenmar 

OS in use:
```
C:\dev\egeria\git\egeria\open-metadata-test\open-metadata-fvt\access-services-fvt\asset-manager-fvt>git log
commit 443f553f8d60863c404f9abf6b22a78808e0addb (HEAD -> master, origin/master, origin/HEAD)
Merge: 4e6e258e84 1cf80964e0
Author: Graham Wallis <graham_wallis@uk.ibm.com>
Date:   Wed Nov 18 15:24:24 2020 +0000
    Merge pull request #4114 from grahamwallis/issue_4112
    Rex enhancements
00:17
java version "1.8.0_241"
Java(TM) SE Runtime Environment (build 1.8.0_241-b07)
Java HotSpot(TM) 64-Bit Server VM (build 25.241-b07, mixed mode)
```
cc: @mandy-chessell @davidradl

**Comment by planetf1** [comment]
Note that our PR builds under github actions run FVTs on Linux only. We do not currently run them on Windows - in particular due to much longer durations of 2.5-3x

**Comment by planetf1** [comment]
Building locally on Windows to verify...

**Comment by planetf1** [comment]
@mandy-chessell as discussed - fvt failure

**Comment by planetf1** [comment]
This seems fixed by #4117 at b4e97cf020779e2da7e6f1bec1ae2222a051a624

**Comment by planetf1** [comment]
Windows merge build now working at https://dev.azure.com/ODPi/Egeria/_build/results?buildId=14260&view=results

@yevgenmar suggest updating from master and retrying. thanks for spotting.
